src/services/blocks/BlocksService.ts

```diff
@@ -28,6 +29,17 @@ import {
 import { isPaysFee } from '../../types/util';
 import { AbstractService } from '../AbstractService';
 
+/**
+ * Types for fetchBlock's options
+ */
+interface FetchBlockOptions {
+	eventDocs: boolean;
+	extrinsicDocs: boolean;
+	checkFinalized: boolean;
+	queryFinalizedHead: boolean;
+	omitFinalizeTag: boolean;
+}
```

Comment:
Can we add some `/** .. */` doc comments for each of these options.

Reply:
```
/**
 * Types for fetchBlock's options
 * @field eventDocs 
 * @field extrinsicDocs
 * @field checkFinalized Option to reduce rpc calls. Equals true when blockId is a hash.
 * @field queryFinalizedHead Option to reduce rpc calls. Equals true when finalized head has not been queried.
 * @field omitFinalizedTag Option to omit the finalized tag, and return it as undefined.
 */
 ```
 
 Any suggestions, I am not sure how to word eventDocs or extrinisicDocs either.